app/javascript/app/pages/country-compare/country-compare-component.jsx

```diff
@@ -46,13 +47,19 @@ const CountryCompare = ({ route, anchorLinks }) => (
           route.sections
             .filter(s => !(isPageContained && s.excludeFromContained))
             .map(section => (
-              <section
+              <Waypoint
+                bottomOffset={'60%'}
+                topOffset={'30%'}
+                onEnter={() => {
+                  setActiveSection(section.hash);
+                }}
+                fireOnRapidScroll={false}
```

Comment:
This still is not working when we select the socioeconomic anchor. Its because the height of the content is too little and it goes back to the second anchor. Can you think about some workaround. Maybe tweaking the topOffset?

Reply:
@Bluesmile82 ready to review ✋